frontend/src/components/identities/IdentityList.vue

```diff
@@ -159,18 +201,36 @@ export default {
     fetchIdentities () {
       identities.getAll().then(resp => {
         this.identities = resp.data
+        this.selected = []
       })
     },
     getMenuItems (item) {
       const result = []
       if (item.type === 'account') {
+        item.possible_actions.forEach(element => {
+          result.push({
+            label: this.$gettext(element.label),
+            icon: element.icon,
+            link: element.url,
+            onClick: () => {
+              window.open(element.url)
```

Comment:
We should find something more generic here. All actions do not require to open a new window